src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -596,7 +597,7 @@ private bool TryGetMentorNode(string dbName, DatabaseTopology topology, ClusterT
             }
             if (_logger.IsInfoEnabled)
             {
-                _logger.Info($"The database {dbName} on {promotable} not ready to be promoted, because the indexes are not up-to-date.\n");
+                _logger.Info($"The database {dbName} on {promotable} not ready to be promoted, because the indexes are not up-to-date." + Environment.NewLine);
```

Comment:
Change to:
The database '{dbName}' on node {promotable} is not ready to be promoted, because the indexes are not up-to-date.